### PR TITLE
Rocketsetup

### PR DESCRIPTION
**Pull request by Abdona**
In This PR, I create the Rocket page main components Structure and tested the UI with pre-defined input.

- CSS Modular approach is used to style the components.
- Create a re used Rocket Component.
- Create a re used Button Component.
- Create The Redux Store actions and Reduceres for the Rocket Component.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const Navbar = () => (
     </div>
     <div className="Navigation-Links">
       <ul className="Link-List">
+        <li><NavLink to="/" activeClassName="Link-Active" className="Link-NoActive">Rockets</NavLink></li>
```
Comment (Leandro-Barretoo) [inline src/components/Navbar.js]: When I click in the other links the Rockets links stays active. Kindly fix that.
One way you can try to achieve this is by adding exact to NavLink. For example:


`<NavLink exact to="/"></NavLInk>`.


review
 
Thank you

**Review by Leandro-Barretoo — CHANGES_REQUESTED** [review]
Suggestion:
 - For the font family use Poppins from google fonts. Its the most similar to the original template.

File: src/components/Navbar.js (hunk 1 above)
Comment (Abdona) [reply src/components/Navbar.js]: Thank you for your feedback.
The problem is resolved.

**Review by Leandro-Barretoo — APPROVED** [review]
Great job implementing the changes :100: :100: 
The project is complete, you can merge it :heavy_check_mark: :heavy_check_mark:

**Pull request merged**